packages/components/src/cookies_form/index.jsx

```diff
@@ -33,6 +33,7 @@ const CookiesForm = ({
   if (compactView) {
     alternativeAction = (
       <span
+        data-track="cookie_form-button_primary"
```

Comment:
https://github.com/goodhood-eu/goodhood/pull/35/files

we changed the same thing